.github/PULL_REQUEST_TEMPLATE/control_plane_template.md

```diff
@@ -39,10 +39,8 @@ If the automation determines you have breaking changes, i.e. Step 1 from the dia
 [1] [ARM review queue] (for **merge** queues, see [4])  
 The PRs are processed by time opened, ascending. Your PR may show up on 2nd or later page. 
 If you addressed Step 1 from the diagram and your PR is not showing up in the queue, ensure the label `ARMChangesRequested` 
-is removed from your PR. This should cause the label `WaitForARMFeedback` to be added.  
-[2] https://aka.ms/azsdk/support/specreview-channel  
-[3] [List of SDK breaking changes approvers] in pinned Teams announcement  
```

Comment:
I think you should also remove the definition of `[List of SDK breaking changes approvers]` from the end of this file.

Reply:
Sure, I removed it.